README.md

```diff
@@ -113,3 +113,18 @@ const MyComponent = ({ userId }) => {
 }
 
 ```
+
+## React Native
+
+React Native is not using the localStorage of the web browser. You need to tell unleash to use your specific storage provider.
+The most common storage provider for React Native is [AsyncStorage](https://github.com/react-native-async-storage/async-storage).
+To configure it, add the following property to your configuration object:
```

Comment:
This is great! 🙌🏼  Just a little rewording suggestion for the first two sentences which makes it clearer why you _can't_ just use the default storage provider. I've assumed that it's because React Native doesn't run in a browser, but please let me know if I'm wrong!

```suggestion
Because React Native doesn't run in a web browser, it doesn't have access to the `localStorage` API. Instead, you need to tell Unleash to use your specific storage provider. The most common storage provider for React Native is [AsyncStorage](https://github.com/react-native-async-storage/async-storage).
To configure it, add the following property to your configuration object:
```